src/main/java/org/molgenis/selenium/model/AbstractModel.java

```diff
@@ -40,4 +41,23 @@ public MenuModel menu()
 		return tableRows.stream().map(elt -> elt.findElements(By.cssSelector("td")))
 				.map(tds -> transform(tds, WebElement::getText)).collect(toList());
 	}
+
+	/**
+	 * Test if an element exists
+	 * 
+	 * @param webDriver
+	 *            WebDriver
+	 * @param context
+	 *            WebElement: can be null than the webDriver is used to search.
+	 * @param by
+	 *            By: by is used to fined the WebElement and define if exist
+	 * @return
+	 */
+	public static boolean exists(WebDriver webDriver, By context, By by)
+	{
+		webDriver.manage().timeouts().implicitlyWait(0, TimeUnit.MILLISECONDS);
+		boolean exists = (null == context ? webDriver : webDriver.findElement(context)).findElements(by).size() != 0;
+		webDriver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS); // Restore default value
+		return exists;
+	}
```

Comment:
Wouldn't you want to wait a bit when you check if something exists?

Since this method is only used in ```assertFalse()``` statements to verify an element doesn't exist, would it make more sense to change this method to ```notExists()```?